src/WebJobs.Script/OutOfProc/Http/HttpWorkerConstants.cs

```diff
@@ -5,7 +5,12 @@ namespace Microsoft.Azure.WebJobs.Script.OutOfProc
 {
     public static class HttpWorkerConstants
     {
+        // Headers
         public const string InvocationIdHeaderName = "X-Azure-Functions-InvocationId";
-        public const string HostVersionHeader = "X-Azure-Functions-HostVersion";
+        public const string HostVersionHeaderName = "X-Azure-Functions-HostVersion";
+        public const string UserAgentHeaderValue = "Azure-Functions-Host";
+
+        // Child Process Env vars
+        public const string PortEnvVarName = "FUNCTIONS_WORKER_PORT";
```

Comment:
Wonder if this should be named FUNCTIONS_HTTPWORKER_PORT so it doesn't get confused with language workers?

Reply:
done!